sig-windows/annual-report-2022.md

```diff
@@ -24,86 +25,87 @@
     - [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support) - v1.26
     - [2802 - Identify Pod's OS during API Server admission](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission) - v1.25
 
-
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   Most of the SIG-Windows sub-projects do not have enough contributors
```

Comment:
I would just say `The SIG-Windows sub-projects...` given that is case across all of them.